server/.env

```diff
@@ -1 +1,3 @@
-MONGODB_URI="mongodb+srv://emanueldrusu:Mongowebgurus22@wgclusterlearn.gr0pdlo.mongodb.net/?retryWrites=true&w=majority"
\ No newline at end of file
+MONGO_URL = "mongodb+srv://biroaron:123@cluster0.y7nbxkk.mongodb.net/ToDo?retryWrites=true&w=majority"
```

Comment:
A better approach here is to have different variables for user and password like :
MONGO_USER=
MONGO_PASSWORD=
Then you build your connection string in the file where you use it